xapian-applications/omega/worker.cc

```diff
@@ -176,14 +176,16 @@ Worker::extract(const std::string& filename,
 		return true;
 	    case MSG_NON_FATAL_ERROR:
 		if (strstate.length() > 1) {
-		    error = (strstate.c_str() + 1);
+		    error.assign(strstate, 1, string::npos);
+		} else {
+		    error = "Couldn't extract text from " + filename;
 		}
 		return false;
 	    default:
 		break;
 	}
     }
-    error = "The assistant process " + filter_module + " fails";
+    error = "Fatal Error: The assistant process " + filter_module + " failed";
```

Comment:
The error is fatal for the assistant, not for the main omindex process.

If we add "Fatal Error" here, when omindex reports the error message to the user it'll look confusing because omindex is continuing to run despite having reported a fatal error.

Reply:
Okay, I will remove that.